src/utils/extractFileSchemeFromObject.ts

```diff
@@ -0,0 +1,12 @@
+import { FileObject } from "../lib/types/FileObject.js";
+
+export function extractFileSchemeFromObject(fileObject: FileObject) {
+  return {
+    _id: fileObject._id,
+    file_path: fileObject.file_path,
+    extension: fileObject.extension,
+    url_path: fileObject.url_path,
+    filetype: fileObject.filetype,
+    metadata: fileObject.metadata,
+  };
+}
```

Comment:
As I mentioned before, I'd move "marshalling" responsibility to relevant Mddb* classes and do it only right before inserting into db.